### ISSUE TITLE
Implement Vercel Accessibility Audit Results

### ISSUE THREAD

**Issue by HansenDJ** [ClearsiteConsultants/client-finder-portal#7]
### ARIA hidden element must not be focusable or contain focusable elements
- Ensures aria-hidden elements are not focusable nor contain focusable elements. [Learn More](https://dequeuniversity.com/rules/axe/4.9/aria-hidden-focus?application=Vercel%20Toolbar)
- `.min-h-screen.flex-col[data-aria-hidden="true"]`

### Document should have one main landmark
- Ensures the document has a main landmark. [Learn More](https://dequeuniversity.com/rules/axe/4.9/landmark-one-main?application=Vercel%20Toolbar)
- `html`
